multi_imbalance/ensemble/soup_bagging.py

```diff
@@ -48,28 +51,41 @@ def __init__(self, classifier=None, maj_int_min=None, n_classifiers=5):
                 self.classifiers.append(KNeighborsClassifier())
 
     @staticmethod
-    def fit_classifier(args):
+    def fit_classifier(args: list) -> Tuple[Any, np.ndarray]:
```

Comment:
list type should be from typing library, could Any be replaced by something more expressive, like
`def fit_classifier(args: List) -> Tuple[BaseClassifier, np.ndarray]:`?